CSS calc(): clarify units usage (#30457)
mdn/content: files/en-us/web/css/calc/index.md

Reviewer: A few suggestions:

1. Let's introduce the term "operand" in the paragraph after the syntax:

"The calc() function takes a single expression as its parameter, and the expression's result is used as the value for a CSS property. In this expression, operands, which are the values being operated on, can be combined using the operators listed below. When the expression contains multiple operands, standard operator precedence rules are followed to derive the result."

2. Let's also expand the description of the operators. For example:

```
    - `+`
        - : Adds the specified operands.
    - `-`
        - : Subtracts the second operand from the first operand.
 ```

width: calc(100% - 80px);
```

> The `calc()` function takes a single expression as its parameter, and the expression's result is used as the value for a CSS property. In this expression, the [operands](/en-US/docs/Glossary/Operand) can be combined using the [operators](/en-US/docs/Glossary/Operator) listed below. When the expression contains multiple operands,`calc()` uses the standard [operator precedence rules](/en-US/docs/Learn/JavaScript/First_steps/Math#operator_precedence):

Reviewer: ```suggestion
The `calc()` function takes a single expression as its parameter, and the expression's result is used as the value for a CSS property. In this expression, the {{Glossary("operand", "operands")}} can be combined using the {{Glossary("operator", "operators")}} listed below. When the expression contains multiple operands,`calc()` uses the standard [operator precedence rules](/en-US/docs/Learn/JavaScript/First_steps/Math#operator_precedence):
```

- `-`
  - : Subtraction.
- `*`

>   - : Multiplication. At least one of the operands must be a {{cssxref("&lt;number&gt;")}}.

Reviewer: ```suggestion
  - : Multiplies the specified operands. At least one of the operands must be a {{cssxref("&lt;number&gt;")}}.
```

- `/`

>   - : Division. The right-hand side must be a {{cssxref("&lt;number&gt;")}}.

Reviewer: ```suggestion
  - : Divides the left-side operand (dividend) by the right-side operand (divisor). The right-hand operand, the divisor, must be a {{cssxref("&lt;number&gt;")}}.
```

> All operands must be suffixed with an appropriate units string, except those that "must be a {{cssxref("&lt;number&gt;")}}". You can use different units for each value in your expression, if you wish. You may also use parentheses to establish computation order when needed.

Reviewer: ```suggestion
All operands, except of the type {{cssxref("&lt;number&gt;")}}, must be suffixed with an appropriate unit string, such as `px`, `em`, or `%`. You can use a different unit with each each operand in your expression. You may also use parentheses to establish computation order when needed.
```
I think we can move this paragraph up before the description of the operators.
